src/ts-main.ts

```diff
@@ -0,0 +1,149 @@
+import { LitElement, css, html } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { marked } from 'marked';
+
+import './ts-section.ts';
+
+const githubApi =
+  'https://api.github.com/users/trustedshops-public/repos?per_page=100&type=owner';
+
+type GroupRepository = Map<string, Repository[]>;
+
+export type Repository = {
+  id: number;
+  name: string;
+  html_url: string;
+  description: string;
+  created_at: string;
+  updated_at: string;
+  topics: string[];
+  [key: string]: unknown;
+};
+
+const getOrderedRepositories = (list: Repository[]): GroupRepository => {
+  const data: GroupRepository = new Map();
+
+  list.forEach((item) => {
+    const [topic] = item.topics.filter((topic: string) =>
+      topic.startsWith('ts')
+    );
+
+    if (!topic) {
+      return;
+    }
+
+    const title = topic.substring(2);
+
+    data.has(title)
+      ? data.set(title, [...data.get(title)!, item])
+      : data.set(title, [item]);
+  });
+  return data;
+};
+
+@customElement('ts-main')
+export class TsMain extends LitElement {
+  @state()
+  private data: GroupRepository = new Map();
+
+  @state()
+  intro: string = '';
+
+  async connectedCallback(): Promise<void> {
+    super.connectedCallback();
+    const response = await fetch(githubApi);
+    const rawList: Repository[] = await response.json();
```

Comment:
suggestion: Add very basic error handling just in case GitHub API rate limit is reached etc. E. g. an simple alert telling "Hey we were not able to fetch our projects, please try again later"

For future version could also be a nice feature to cache the data in IndexedDB or something. But that's totally optional.